source/parser/riscv/riscv-parser.cpp

```diff
@@ -225,11 +214,10 @@ const SyntaxInformation RiscvParser::getSyntaxInformation() {
   info.addSyntaxRegex("^\\s*\\w+:", SyntaxInformation::Token::Label);
```

Comment:
Might this not be able to be applied rather to all words? (including constants) Or what do you think of this?